velox/docs/functions/spark/string.rst

```diff
@@ -188,6 +188,12 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT rtrim('kr', 'spark'); -- "spa"
 
+.. spark:function:: soundex(string) -> string
+
+    Returns Soundex code of the string. ::
```

Comment:
Maybe better to document the corner cases like input is empty and input string is not character.